portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,24 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.image-thumbnails {
```

Comment:
Are these classes used? Sorry if I'm missing it.

Reply:
Oh that was me figuring out how the css stuff worked via the example. I forgot to remove them.